documentation/smg/07_Windows_Anvil_Node.md

```diff
@@ -0,0 +1,23 @@
+# Windows Anvil Node
```

Comment:
can you add:
- the name of the build node as it appears in Anvil somewhere `PACE Windows (Private)`
- for completeness (in case your Download folder is emptied) the java command to launch the agent (as in the .bat file).